### DISCUSSION TITLE
How to authenticate with Github?

### DISCUSSION THREAD

**Discussion by KittichoteKamalapirat** [invertase/react-native-firebase#7802] [Q&A]
I don't see any mention of authenticating with Github in the [doc](https://rnfirebase.io/auth/usage). Does anyone know how to do it?

**Comment by JohnKesko** [comment]
I was looking for that as well, but since I couldn't find anything, I implemented my own using Expo built in WebBrowser.maybeCompleteAuthSession(); that does the heavy lifting.
https://docs.expo.dev/guides/authentication/

You get the the authorization code back, then you need to implement the rest to handle the tokens etc. I did it with firebase functions.
